projects/unix/Makefile

```diff
@@ -137,6 +137,14 @@ ifeq ($(OS), OSX)
   endif
 endif
 
+# test for essential build dependencies
+ifeq ($(origin PKG_CONFIG), undefined)
+  PKG_CONFIG = $(CROSS_COMPILE)pkg-config
+  ifeq ($(shell which $(PKG_CONFIG) 2>/dev/null),)
```

Comment:
Please don't use this. "which" is less likely to be installed than "pkg-config".

"command -v" is a portable (POSIX mandated) equivalent to "which".

Reply:
I'll see about improving the check for mupen64plus-core in the near future and then change it here too depending on their response.